tests/fixtures/app_data/oai_sets.yaml

```
@@ -0,0 +1,3 @@
+- name: "Set1"
+  spec: "set1"
+  search_pattern: "resource_type.type:dataset"
```

```suggestion
  search_pattern: "metadata.resource_type.id:dataset"
```

minor: just for correctness in case someone greps around for how to filter by resource type :)